packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -1154,6 +1158,16 @@ class SliverAppBar extends StatefulWidget {
   /// Defaults to [NavigationToolbar.kMiddleSpacing].
   final double titleSpacing;
 
+  /// The size of the app bar when it is collapsed.
+  ///
+  /// By default, the collapsed height is set to [kToolbarHeight]. If [bottom]
+  /// is set, its [bottom.preferredSize.height] is added to the height. If
+  /// [primary] is true, any top padding from [MediaQuery] is added as well.
+  ///
+  /// If [pinned] and [floating] is true, with [bottom] set, the collapsed
```

Comment:
is => are

I assume that you're describing the default collapsed height here. So "the collapsed" => "the default collapsed"

Reply:
The `collapsedHeight` is actually ignored when `pinned` and `floating` are true and `bottom` is set. See line 1330 in `app_bart.dart`.

But I should respect it, and default it to `0.0` if `collapsedHeight` is not set. I'll do another revision for this, it will require me to update the `toolbarOpacity` calculation though.